github-lab-bundle-spec.md

```diff
@@ -85,9 +85,6 @@ duration: 60
 level: intro
 tags: [sample, life-changing, gcp]
 
-# The primary instruction content for this lab
```

Comment:
So, this is not configurable anymore. We just require creators to place their instructions under a folder called `instructions`? If so, I am curious as to what caused this change.

Reply:
it is not a change, it's always been like this. GW always rewrites instruction block itself regardless of what you put there. Writing yourself is noop at best, or sometimes confusing because removing an instruction there won't actually remove it